index.html

```diff
@@ -0,0 +1,146 @@
+                </div>
+
+            </section>
+            <section class="scroll-item mid">
+                <div class="category-head">
+                   <h3> Quiz-Miniac </h3>
+                 <a href="./Instruction/instruction.html">Rules</a>
+                </div>
+                <div class="category-cont">
+                    <div class="category-subhead">
+                        <h3>Categories</h3>
+                    </div>
+                    <div class="category-menu">
+                        <div class="menu-item">Anime</div>
+                        <div class="menu-item">English-Series</div>
+                        <div class="menu-item">Bollywood</div>
+                        <div class="menu-item">Hollywood</div>
+                    </div>
+                    <div class="category-item-cards">
+                        <div class="card-item">
+                            <div class="img-cont">
+                                <img src="https://www.imagedecor.com.au/wp-content/uploads/2018/04/PP32769_f.jpg" alt="" srcset="">
+                            </div>
+                            <div class="btn-cont">
+                                <a href="/Instruction/instruction.html">Play</a>
+                            </div>
+                        </div>
+                        <div class="card-item">
+                            <div class="img-cont">
+                                <img src="https://www.imagedecor.com.au/wp-content/uploads/2018/04/PP32769_f.jpg" alt="" srcset="">
+                            </div>
+                            <div class="btn-cont">
+                                <a href="/Instruction/instruction.html">Play</a>
+                            </div>
+                        </div>
+                        <div class="card-item">
+                            <div class="img-cont">
+                                <img src="https://www.imagedecor.com.au/wp-content/uploads/2018/04/PP32769_f.jpg" alt="" srcset="">
+                            </div>
+                            <div class="btn-cont">
+                                <a href="/Instruction/instruction.html">Play</a>
+                            </div>
+                        </div>
+                        <div class="card-item">
+                            <div class="img-cont">
+                                <img src="https://www.imagedecor.com.au/wp-content/uploads/2018/04/PP32769_f.jpg" alt="" srcset="">
+                            </div>
+                            <div class="btn-cont">
+                                <a href="/Instruction/instruction.html">Play</a>
+                            </div>
+                        </div>
+                        
+                    </div>
+                </div>
+            </section>
+            <section class="scroll-item mid-score">
+                <div class="score-head">
+                    <h3>Top Scores</h3>
+                </div>
```

Comment:
again wrapping h3 inside a div is unnecessary, just give the class to h3 and remove the div